app/api/routes.py

```diff
@@ -217,7 +217,7 @@ def get_resources():
         )
 
     # Filter on paid
-    if isinstance(paid, str):
+    if isinstance(paid, str) and paid in ['true', 'false']:
         paidAsBool = paid.lower() == 'true'
```

Comment:
Can we move the `lower()` function call up to the above line so that this still returns all true resources? `/api/v1/resources?paid=TRUE`

We should probably add a unit test around the case sensitivity of all the parameters that we have `lower()` calls like this.

Reply:
I updated the behavior and test. Also updated the /update test to also catch that